addons/website_event_track/static/src/interactions/event_track_reminder.js

```diff
@@ -1,87 +1,90 @@
 import { debounce } from "@web/core/utils/timing";
-import publicWidget from "@web/legacy/js/public/public_widget";
 import { _t } from "@web/core/l10n/translation";
 import { rpc } from "@web/core/network/rpc";
 import { Component } from "@odoo/owl";
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
 
-publicWidget.registry.websiteEventTrackReminder = publicWidget.Widget.extend({
-    selector: '.o_wetrack_js_reminder',
-    events: {
-        'click': '_onReminderToggleClick',
-    },
+export class WebsiteEventTrackReminder extends Interaction {
+    static selector = ".o_wetrack_js_reminder";
 
-    /**
-     * @override
-     */
-    init: function () {
-        this._super.apply(this, arguments);
-        this._onReminderToggleClick = debounce(this._onReminderToggleClick, 500, true);
-        this.notification = this.bindService("notification");
-    },
+    dynamicContent = {
+        _root: { "t-on-click": (ev) => this._onReminderToggleClick(ev) },
+    };
 
-    //--------------------------------------------------------------------------
-    // Handlers
-    //-------------------------------------------------------------------------
+    setup() {
+        this._onReminderToggleClick = debounce(
+            this._onReminderToggleClick,
+            500,
+            true
+        );
+    }
 
-    /**
-     * @private
-     * @param {Event} ev
-     */
-    _onReminderToggleClick: function (ev) {
+    _onReminderToggleClick(ev) {
         ev.stopPropagation();
         ev.preventDefault();
```

Comment:
You can replace these two lines by adding `.prevent.stop` on the `t-on-click` declaration.